new-lamassu-admin/src/pages/LoyaltyPanel/CouponCodes.styles.js

```diff
@@ -0,0 +1,37 @@
+import { spacer, fontPrimary, primaryColor } from 'src/styling/variables'
+
+export default {
```

Comment:
New linter doesn't allow anonymous exports. Make the styles into its own variable and then export them at the end.
for example, 
const styles = {...}
export default styles